### PR TITLE
added extra registry keys

### PR DESCRIPTION
**Pull request by RAJU2529**
added extra registry keys taken reference from windows 10 1909 enterprise

registry

### CODE REVIEW

**Comment by e0i** [comment]
@JohanFreelancer9 Minor style checks needed for this PR. Thanks.

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 1]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
 
 All Capability SIDs that the operating system is aware of are stored in the Windows Registry in the path `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities'. Any Capability SID added to Windows by first or third-party applications will be added to this location.
 
+## Example  registry keys are  taken from windows 10 1909 64bit Enterprise edition
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
## Example—registry keys taken from Windows 10, version 1909, 64-bit Enterprise edition
```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 2]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+ You may see the following registry keys are under AllCachedCapabilities
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
You may see the following registry keys under AllCachedCapabilities:

```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 3]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_DevUnlock
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_DevUnlock

```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 4]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_DevUnlock_Internal
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_DevUnlock_Internal

```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 5]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_Enterprise
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_Enterprise

```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 6]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_General
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_General

```

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 7]
```diff
@@ -289,6 +289,15 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
...
+HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_Restricted
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities\capabilityClass_Restricted

```

**Comment by RAJU2529** [comment]
@e0i . committed changes

**Comment by illfated** [comment]
@JohanFreelancer9 : I must admit that from my perspective, the em dash looks a bit odd when it is not used to insert an embedded clause. I can hardly remember seeing this form in use, but if you are certain about this being the correct usage, I will not complain.

File: windows/security/identity-protection/access-control/security-identifiers.md [hunk 8]
```diff
@@ -289,6 +289,16 @@ Capability Security Identifiers (SIDs) are used to uniquely and immutably identi
 
 All Capability SIDs that the operating system is aware of are stored in the Windows Registry in the path `HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\SecurityManager\CapabilityClasses\AllCachedCapabilities'. Any Capability SID added to Windows by first or third-party applications will be added to this location.
 
+## Example—registry keys taken from Windows 10, version 1909, 64-bit Enterprise edition
```
Comment (JohanFreelancer9) [inline windows/security/identity-protection/access-control/security-identifiers.md]:
```suggestion
## Examples of registry keys taken from Windows 10, version 1909, 64-bit Enterprise edition
```

**Comment by illfated** [comment]
(Ref. #5745 )

**Pull request merged**